client/src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -102,14 +102,6 @@ Breadcrumb.propTypes = {
   crumbs: PropTypes.arrayOf(PropTypes.shape({
     onClick: PropTypes.func,
     text: PropTypes.string,
-    // pass in a single icon (retain backwards compatibility)
-    icon: PropTypes.shape({
-      nodeName: PropTypes.string,
-      className: PropTypes.string,
-      onClick: PropTypes.func,
-      action: (props) => { if (props.action) { throw new Error('action: no longer used'); } },
-    }),
```

Comment:
This isn't used anywhere that I can find - and if anyone's using it for their own uses they can just pass it as an array to the `icons` prop instead.